process-engine/bpmn-studio
# ✨ Include Process Solution Explorer Width in Property Panel MaxWidth Calculation

## What did you change?

This PR adds the width of the process solution explorer to the calculation of the maxWidth of the property panel.
When hiding/showing the process solution explorer the property panel should be hidden/showed when there is/was not enough space.

This PR also refactors the resize functionality.

Fixes #416 
Fixes #459 
Closes #463 

## How can others test the changes?

- Open the Bpmn-Studio
- Go to the Detail View of any ProcessDefinition
- Resize the Property Panel and Hide/Show the Process Solution Explorer
- See that the Process Solution Explorer can not break the Property Panel anymore

## PR-Checklist

Please check the boxes in this list after submitting your PR:
